src/main/java/org/jenkinsci/plugins/rundeck/cache/InMemoryRundeckJobCache.java

```diff
@@ -97,26 +84,16 @@ public void invalidate() {
         rundeckJobInstanceAwareCache.invalidateAll();
     }
 
+    @SuppressFBWarnings(value = "NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE")
     private JobItem findByJobIdInCacheOrAskServer(final String rundeckJobId, String rundeckInstanceName,
-                                                  final RundeckManager rundeckInstance) throws ExecutionException {
+                                                  final RundeckManager rundeckInstance) {
         Cache<String, JobItem> rundeckJobCache = rundeckJobInstanceAwareCache.get(rundeckInstanceName);
 
-        JobItem tmp = rundeckJobCache.get(rundeckJobId, new Callable<JobItem>() {
-            public JobItem call() throws Exception {
-                return RundeckNotifier.RundeckDescriptor.findJobUncached(rundeckJobId, rundeckInstance);
-            }
-        });
+        JobItem tmp = rundeckJobCache.get(rundeckJobId, unused -> RundeckNotifier.RundeckDescriptor.findJobUncached(rundeckJobId, rundeckInstance));
         logCacheStatsIfAppropriate(rundeckInstanceName, rundeckJobCache);
         return tmp;
```

Comment:
I have no idea why I named it `tmp` back in 2016 :smile: . Probably, I intended to rename it later to `foundJob` (or something), but forgot. Nevertheless, it is not related to your refactoring.